test_runner/regress/test_storage_controller.py

```diff
@@ -466,6 +486,69 @@ def test_storage_controller_onboarding(neon_env_builder: NeonEnvBuilder, warm_up
     env.storage_controller.consistency_check()
 
 
+def test_storage_controller_onboard_detached(neon_env_builder: NeonEnvBuilder):
```

Comment:
Can this test run on the default pg version only?

Reply:
it sure can.  added decorator